data/sql/updates/pending_db_world/rev_1709209411982077500.sql

```diff
@@ -0,0 +1,34 @@
+(@ENTRY, 0, 1, 0, 106, 0, 100, 0, 24000, 28000, 24000, 28000, 11, 51806, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Event 106 has invalid Readable format in events.json - Self: Cast spell  Power Flux (51806) with flags interrupt previous on Self'),
+(@ENTRY, 0, 2, 0, 0, 0, 100, 0, 7000, 12000, 14000, 22000, 11, 51804, 32, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 'Every 14 - 22 seconds (7 - 12s initially) (IC) - Self: Cast spell  Power Siphon (51804) with flags aura not present on Random hostile'),
+(@ENTRY, 0, 3, 0, 0, 0, 100, 0, 9000, 12000, 20000, 25000, 11, 51808, 1, 0, 0, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, 'Every 20 - 25 seconds (9 - 12s initially) (IC) - Self: Cast spell  Fury of the Blue (51808) with flags interrupt previous on Random hostile (not top)'),
+(@ENTRY, 0, 4, 5, 2, 0, 100, 0, 0, 30, 120000, 130000, 11, 51800, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'When health between 0%-30%% (cooldown 120000 - 130000 ms) - Self: Cast spell  Might of Malygos (51800) on Victim'),
+(@ENTRY, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'When health between 0%-30%% (cooldown 120000 - 130000 ms) - Self: Talk You test my patience, you now see the true might of the Blue. (0) to invoker'),
+(@ENTRY, 0, 6, 0, 60, 0, 100, 0, 106800, 160000, 135600, 172900, 80, 2682800, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Every 135.6 - 172.9 seconds (106.8 - 160s initially) - Self: Start timed action list id #Magister Keldonus #0 (2682800) (update out of combat) // -inline'),
+(@ENTRY * 100, 9, 0, 0, 0, 0, 100, 0, 0, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'After 0 seconds - Self: Talk My lady, the surge needle is fully operational. (1) to None'),
+(@ENTRY * 100, 9, 1, 0, 0, 0, 100, 0, 4500, 4500, 0, 0, 1, 2, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'After 4.5 seconds - Self: Talk The ley line beneath this night elf temple is now flowing in... (2) to None'),
+(@ENTRY * 100, 9, 2, 0, 0, 0, 100, 0, 6400, 6400, 0, 0, 1, 0, 0, 0, 0, 0, 0, 19, 26832, 15, 0, 0, 0, 0, 0, 'After 6.4 seconds - Closest alive creature Grand Magus Telestra (26832) in 15 yards: Talk You have done well, Keldonus. Lord Malygos will be quite ple... (0) to invoker'),
+(@ENTRY * 100, 9, 3, 0, 0, 0, 100, 0, 5000, 5000, 0, 0, 1, 3, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'After 5 seconds - Self: Talk I live only to serve the master, my lady. (3) to Self'),
+(@ENTRY, 0, 7, 0, 6, 0, 100, 0, 0, 0, 0, 0, 1, 2, 0, 0, 0, 0, 0, 19, 26832, 15, 0, 0, 0, 0, 0, 'On death - Closest alive creature Grand Magus Telestra (26832) in 15 yards: Talk This transgression against the Blue Dragonflight has been no... (2) to invoker'),
+(@ENTRY, 0, 8, 0, 4, 0, 100, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 19, 26832, 15, 0, 0, 0, 0, 0, 'On aggro - Closest alive creature Grand Magus Telestra (26832) in 15 yards: Talk Deal with this interruption, Keldonus. After you are through... (1) to invoker');
```

Comment:
comments are too long. you don't need to put half of the text line in there. Just do something like "Say - line x" or something similar